DataFormats/HGCalReco/interface/Trackster.h

```diff
@@ -24,6 +24,23 @@ namespace ticl {
     // the first element is on the inner layer and
     // the outer element is on the outer layer.
     std::vector<std::array<unsigned int, 2> > edges;
+
+    // regressed energy
+    float regressed_energy;
+
+    // trackster ID probabilities
+    std::array<float, 5> id_probabilities;
```

Comment:
Sorry for being late.

We do have enum ParticleType defined in: https://cmssdt.cern.ch/lxr/source/DataFormats/ParticleFlowCandidate/interface/PFCandidate.h
I haven't looked at this PR closely, but I believe it will help if TICL's particle ID uses the same convention when we try to integrate TICL in PF as a whole.

Reply:
Thanks for the reference.

We will probably need the two additional classes (`unknown` and `ambiguous`) internally in  TICL for further studies. However, there will be a PFTICLProducer (@steggema) that is fully compliant with the ParticleType enum.

Reply:
OK. I will keep eyes on it when it comes out.